packages/trees/frontend/sirius-components-trees/src/views/ExplorerView.tsx

```diff
@@ -84,7 +85,14 @@ const useExplorerViewStyles = makeStyles((theme) => ({
   },
 }));
 
-export const ExplorerView = ({ editingContextId, selection, setSelection, readOnly }: WorkbenchViewComponentProps) => {
+export const ExplorerView = ({
```

Comment:
Shouldn't it be split in two different components, `TreeView` and `ExplorerView`?

Reply:
done.